org.eclipse.dawnsci.nexus/src/org/eclipse/dawnsci/nexus/ServiceHolder.java

```diff
@@ -20,7 +20,7 @@ public static INexusFileFactory getNexusFileFactory() {
 		return nexusFileFactory;
 	}
 	
-	public static void setNexusFileFactory(INexusFileFactory nexusFileFactory) {
+	public void setNexusFileFactory(INexusFileFactory nexusFileFactory) {
 		ServiceHolder.nexusFileFactory = nexusFileFactory;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)